src/components/charts/SummaryPieChart/index.jsx

```diff
@@ -181,7 +180,6 @@ SummaryPieChart.defaultProps = {
   useCustomizedColorMap: false,
   customizedColorMap: ['#3283c8'],
   maximumDisplayItem: 15,
-  chartIsEmpty: false,
   chartEmptyMessage: 'Cannot render this chart because some fields don\'t apply',
```

Comment:
suggest to change this default message to something shorter, like "This chart is empty" or "No data"

you can verify how it looks in the storybook